c_pal_ll/interfaces/inc/c_pal/interlocked_macros.h

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#ifndef INTERLOCKED_MACROS_H
+#define INTERLOCKED_MACROS_H
```

Comment:
I vote for this header to be included directly in #include "c_pal/interlocked.h" :)

Reply:
sure, one less include to add everywhere else